src/java.base/share/classes/java/lang/Class.java

```diff
@@ -352,9 +352,9 @@ static String typeVarBounds(TypeVariable<?> typeVar) {
      * runtime {@code Class} descriptor for the class named
      * {@code java.lang.Thread}:
      *
-     * <blockquote>
-     *   {@code Class t = Class.forName("java.lang.Thread")}
-     * </blockquote>
+     * {@snippet lang="java" :
+     *   Class<?> t = Class.forName("java.lang.Thread")
```

Comment:
This should have a final semicolon, as it's already a variable declaration and initialization.